docs/platforms/dotnet/common/profiling/index.mdx

```diff
@@ -31,43 +31,43 @@ dotnet add package Sentry.Profiling --prerelease
 
 Profiling depends on Sentry’s performance monitoring product being enabled beforehand. To enable performance monitoring in the SDK, set the `TracesSampleRate` option to the desired value.
 
-```csharp {tabTitle:iOS/MacCatalyst}
+```csharp {tabTitle:Windows/Linux/macOS}
 SentrySdk.Init(options =>
 {
     // ... usual setup options omitted for clarity (see Getting Started) ...
 
     // Sample rate for your transactions, e.g. value 0.1 means we want to report 10% of transactions.
+    // Setting 1.0 means all transactions are profiled.
     // We recommend adjusting this value in production.
     options.TracesSampleRate = 1.0;
 
     // Sample rate for profiling, applied on top of othe TracesSampleRate,
     // e.g. 0.2 means we want to profile 20 % of the captured transactions.
     // We recommend adjusting this value in production.
     options.ProfilesSampleRate = 1.0;
+
+    // Requires NuGet package: Sentry.Profiling
+    // Note: by default, the profiler is initialized asynchronously, this can be tuned by passing an desired initialization timeout to the constructor.
+    options.AddIntegration(new ProfilingIntegration(
+        // During startup, wait up to 500ms to profile the app startup code. This could make launching the app a bit slower so comment it out if your prefer profiling to start asynchronously
+        TimeSpan.FromMilliseconds(500)
+    ));
 });
 ```
 
-```csharp {tabTitle:Windows/Linux/macOS}
+```csharp {tabTitle:iOS/MacCatalyst}
 SentrySdk.Init(options =>
 {
     // ... usual setup options omitted for clarity (see Getting Started) ...
 
     // Sample rate for your transactions, e.g. value 0.1 means we want to report 10% of transactions.
-    // Setting 1.0 means all transactions are profiled.
```

Comment:
Not sure why this line was removed. Can we add it back in so the two code samples have it?